src/lib/simulation/toa.ts

```diff
@@ -1155,46 +1169,75 @@ export async function toaStartCommand(
 			}
 		};
 	}
-
-	const {
-		duration,
-		totalReduction,
-		reductions,
-		wipedRoom: _wipedRoom,
-		deathDuration,
-		parsedTeam,
-		messages
-	} = await createTOATeam({
-		team: users,
-		raidLevel
+	const toaSimUsers = await Promise.all(
+		users.map(async i => ({ user: i, userStats: await i.fetchStats(), minigameScores: await i.fetchMinigames() }))
+	);
+	const baseDuration = createTOATeam({
+		team: toaSimUsers,
+		raidLevel,
+		quantity: 1
+	})[0].duration;
+	const maxTripLength = Time.Hour * 2; // Math.max(...users.map(i => calcMaxTripLength(i, activity_type_enum.TombsOfAmascut)));
+	const maxQuantity = clamp(Math.floor(maxTripLength / baseDuration), 1, 5);
+	const quantity = clamp(quantityInput ?? 1, 1, maxQuantity);
```

Comment:
shouldn't this be
```ts
const quantity = clamp(quantityInput ?? maxQuantity, 1, maxQuantity);
```
so that it automatically chooses the max number possible and you don't have to guess